praw/models/reddit/subreddit.py

```diff
@@ -1881,6 +1899,193 @@ def update(self, **settings):
         )
 
 
+class SubredditModerationStream:
+    """Provides moderator streams."""
+
+    def __init__(self, subreddit):
+        """Create a SubredditModerationStream instance.
+
+        :param subreddit: The moderated subreddit associated with the streams.
+
+        """
+        self.subreddit = subreddit
+
+    def edited(self, only=None, **stream_options):
+        """Yield edited comments and submissions as they become available.
+
+        :param only: If specified, one of ``'comments'``, or ``'submissions'``
+            to yield only results of that type.
+
+        Keyword arguments are passed to :func:`.stream_generator`.
+
+        For example, to retrieve all new edited submissions/comments made
+        to all moderated subreddits, try:
+
+        .. code:: python
+
+           for item in reddit.subreddit('mod').mod.stream.edited():
+               print(item)
+
+        """
+        return stream_generator(
+            self.subreddit.mod.edited, only=only, **stream_options
+        )
+
+    def log(self, action=None, mod=None, **stream_options):
+        """Yield moderator log entries as they become available.
+
+        :param action: If given, only return log entries for the specified
+            action.
+        :param mod: If given, only return log entries for actions made by the
+            passed in Redditor.
+
+        For example, to retrieve all new mod actions made to all moderated
+        subreddits, try:
+
+        .. code:: python
+
+           for log in reddit.subreddit('mod').mod.stream.log():
+               print("Mod: {}, Subreddit: {}".format(log.mod, log.subreddit))
+
+        """
+        return stream_generator(
+            self.subreddit.mod.log,
+            action=action,
+            mod=mod,
+            attribute_name="id",
+            **stream_options
+        )
+
+    def modmail_conversations(
+        self, other_subreddits=None, sort=None, state=None, **stream_options
+    ):
+        """Yield unread moderator messages as they become available.
```

Comment:
Does this method stream messages from new modmail, old modmail, or both? Can that please be documented?

Reply:
New modmail. I based it on the parent method here: https://github.com/praw-dev/praw/blob/0c198a2eb47e001cc4bec82353f45e826be96c2c/praw/models/reddit/subreddit.py#L2521